.circleci/src/jobs/@backend.yml

```diff
@@ -43,7 +43,7 @@ lint-back:
         command: ~/.local/bin/flake8
     - run:
         name: Lint code with isort
-        command: ~/.local/bin/isort --recursive --check-only .
```

Comment:
so are you sure that it is still recursively checking the whole `.` directory?

Reply:
According to the doc, yes.
https://pycqa.github.io/isort/docs/upgrade_guides/5.0.0/#-recursive-or-rc

> In 5.0.0 directories are automatically traversed for all Python files